unbxdSearch.js

```diff
@@ -1277,16 +1380,36 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
 
             return this;
         },
-        setDefaultParams: function(params) {
+        setDefaultParams: function (params) {
             var newparams, diff;
             var oldparams = JSON.stringify(params);
             this.options.setDefaultFilters.call(this);
+
+            this.params['extra']['version'] = 'V2';
+
+            if (this.options.facetMultilevel) {
+                this.params['extra']['facet.multilevel'] = 'categoryPath';
+                this.params['extra']['f.categoryPath.displayName'] = this.options.facetMultilevelName;
+                this.params['extra']['f.categoryPath.max.depth'] = '6';
+                this.params['extra']['f.categoryPath.facet.limit'] = '100';
+                this.params['extra']['f.categoryPath.facet.version'] = 'V2';
+            }
+
+            if (this.options.variants) {
+                this.params['extra']['variants'] = true;
+                this.params['extra']['variants.count'] = this.options.variantsCount || 1;
+            }
+
+            if (this.options.isSwatches) {
+                this.params['extra']['variants.groupby'] = this.options.mappedFields.variantFields.groupBy;
```

Comment:
can you if `groupby` exists inside variantFields. And explanation for this config is missing in README.

Reply:
Explanation Added in Readme.
groupBy field is to be added by the user, if swatches are required, as it is passed in the url.
Hence the check for swatches should suffice.

Reply:
Done